hack/kubernetes_version/update_kubernetes_version.go

```diff
@@ -77,7 +128,7 @@ func main() {
 			return err
 		}
 		re = regexp.MustCompile(`kubernetesVersion: .*`)
-		cf = []byte(re.ReplaceAllString(string(cf), "kubernetesVersion: "+v))
+		cf = []byte(re.ReplaceAllString(string(cf), "kubernetesVersion: "+vDefaultMM+".0")) // TODO: let <PATCH> version to be the latest one instead of "0"
```

Comment:
Can you clarify this TODO? I think it may be removable.

It's intentional that the tests are using `.0` to avoid unnecessary testdata updates for every minor release.

Reply:
i thought it could be needed sometimes later, but if we would stick always to `.0` minor release, then yes, that TODO is not needed
then also, do we need the testData update section here at all if it is static (ie, miniro release is fixed to `0.`) or shall we remove it altogether?

Reply:
@tstromberg i've submitted a new pr where i removed the 'todo' and left `testData` update section as-is until we clarify it